integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/DataMapListeners.scala

```diff
@@ -65,7 +65,7 @@ object DropDataMapPostListener extends OperationEventListener {
           Some(dataMapSchema.get.getRelationIdentifier.getDatabaseName),
           dataMapSchema.get.getRelationIdentifier.getTableName,
           dropChildTable = true
-        ).run(sparkSession)
+        ).processMetadata(sparkSession)
```

Comment:
I think it is better to move this call to CarbonDropDataMapCommand.processMetadata

Reply:
@jackylk  code is moved to CarbonDropDataMapCommand